apps/dashboard/app/helpers/system_status_helper.rb

```diff
@@ -6,37 +6,27 @@ def title(cluster)
     "#{cluster.metadata.title.titleize} Cluster Status"
   end
 
-  def percent(value, total)
-    "#{(value.to_f / total * 100).round 2}%"
-  end
-
-  def generic_status(active, total, name)
-    free = number_with_delimiter(total - active)
-    "#{name} Active: #{number_with_delimiter(active)}/#{number_with_delimiter(total)} (#{free} Free)"
-  end
-
-  def node_status(cluster_info)
-    generic_status(cluster_info.active_nodes, cluster_info.total_nodes, 'Nodes')
-  end
-
-  def node_pct(cluster_info)
-    percent(cluster_info.active_nodes, cluster_info.total_nodes)
-  end
-
-  def processor_status(cluster_info)
-    generic_status(cluster_info.active_processors, cluster_info.total_processors, 'Processors')
+  def status_hash(name, active, total)
+    {
+      strings: [
+        "#{name} Active: #{number_with_delimiter(active)}",
```

Comment:
Yea I kinda feel like we don't need the # of active things at all.  A researcher's only going to care about what's free (so they can use what's free). 

What's active is interesting as a staff member/admin, but that's not who this page is for.